README.md

```diff
@@ -97,6 +97,20 @@ version. This version number comes from a `yum search kubelet
 ansible-playbook -i inventory/vms.local.generated kube-install.yml -e 'kube_version=1.6.7-0'
 ```
 
+## Install specific binaries
+
+By default, we're installing the Kubelet (and `kubeadm`, `kubectl` and the core CNI plugins) via RPM. However, if you'd like to install specific binaries for either the kubelet, kubeadm or kubetl -- you can do so by specifying that you'd like to perform a binary install and specify URLs (that point to, say, binaries in a GitHub release).
```

Comment:
> install specific binaries for either the Kubelet (this should be proper noun?), `kubeadm`, or `kubectl`, you can do so by...

Reply:
Good spot, I think it shouldn't be proper.